app/helpers/activity_helper.rb

```diff
@@ -33,4 +35,20 @@ def policy_markers_iati_codes_to_enum(code)
   def policy_markers_desertification_iati_codes_to_enum(code)
     Activity::DESERTIFICATION_POLICY_MARKER_CODES.key(code.to_i)
   end
+
+  def benefitting_countries_with_percentages(benefitting_countries)
+    # Get an equal percentage split between all the countries
+    # (together with the remainder if possible)
+    percentage, remainder = 100.divmod(benefitting_countries.count)
```

Comment:
At the moment we haven't added presence validation for `benefitting_countries`, while we have both the new and old fields being collected.

Can this cope with the number being `0`?

Reply:
I've tweaked this to return an empty array if `benefitting_countries` is blank now 👍